### DISCUSSION TITLE
Why it's taking forever to address issues? currently 1.3K open!!!

### DISCUSSION THREAD

**Discussion by mtermoul** [vuetifyjs/vuetify#12281] [General] [answered]
I notice that the it's really taking a long time to even comment on an issue. Not to mention how long would it take to address it. Things use to be more enjoyable when we Vuetify version was 1, 1.5. 
I understand that everyone is busy, but it should take really little time just to take a look at new issues, tag them, and at least give an idea if this going to be a no-fix or stir to the right direction.

**Comment by icleolion** [comment]
Please feel free to give up **_a little_** of your time to join the team and help them triage the issues, if you think that is all it takes.

**Comment by johnleider** [comment, marked as answer]
There are numerous reason for this and is something I will detail in the future. Thank you for your understanding.
